packages/store-api/src/platforms/vtex/resolvers/review.ts

```diff
@@ -0,0 +1,11 @@
+type Resolver = () => unknown
```

Comment:
hmmm, I'm in doubt whether we should have a file for a resolver that doesn't exist.